frontend/javascripts/oxalis/view/statusbar.js

```diff
@@ -0,0 +1,220 @@
+
+type OwnProps = {||};
+type StateProps = {|
+  activeResolution: Vector3,
+  activeViewport: OrthoView,
+  mousePosition: ?Vector2,
+  isSkeletonAnnotation: boolean,
+  activeTool: ?VolumeTool,
+|};
+type Props = {| ...OwnProps, ...StateProps |};
+type State = {||};
+
+const borderToggleButtonMargin = 40;
+const fontSize = 14;
+const spaceBetweenItems = 25;
+const lineColor = "rgba(255, 255, 255, 0.67)";
+
+const statusbarStyle: Object = {
+  marginLeft: borderToggleButtonMargin,
+  marginRight: borderToggleButtonMargin,
+  fontSize,
+  display: "flex",
+  flexWrap: "wrap",
+  overflow: "hidden",
+};
+const defaultIconStyle = { height: fontSize - 1, marginTop: 2 };
+const defaultInfoStyle = { display: "inline-block", textAlign: "left" };
+const defaultShortcutStyle = { marginLeft: spaceBetweenItems };
+
+const hasSegmentation = () => Model.getSegmentationLayer() != null;
+
+class Statusbar extends React.PureComponent<Props, State> {
+  getSegmentationCube(): Cube {
+    const segmentationLayer = Model.getSegmentationLayer();
+    return segmentationLayer.cube;
+  }
+
+  getPosString(pos: Vector3) {
+    return V3.floor(pos).join(",");
+  }
+
+  getZoomShortcut() {
+    return (
+      <span key="zoom" style={defaultShortcutStyle}>
+        <span
+          key="zoom-i"
+          className="keyboard-key-icon-small"
+          style={{ borderColor: lineColor, marginTop: -1 }}
+        >
+          {/* Move text up to vertically center it in the border from keyboard-key-icon-small */}
+          <span style={{ position: "relative", top: -2 }}>Alt</span>
+        </span>{" "}
+        +
+        <img
+          className="keyboard-mouse-icon"
+          src="/assets/images/icon-statusbar-mouse-wheel.svg"
+          alt="Mouse Wheel"
+          style={defaultIconStyle}
+        />
+        Zoom in/out
+      </span>
+    );
+  }
+
+  getShortcuts() {
+    return (
+      <React.Fragment>
+        <span
+          style={{
+            marginLeft: "auto",
+            textTransform: "capitalize",
+          }}
+        >
+          <img
+            className="keyboard-mouse-icon"
+            src={
+              this.props.activeTool === VolumeToolEnum.PICK_CELL ||
+              this.props.activeTool === VolumeToolEnum.FILL_CELL
+                ? "/assets/images/icon-statusbar-mouse-left.svg"
+                : "/assets/images/icon-statusbar-mouse-left-drag.svg"
+            }
+            alt="Mouse Left Drag"
+            style={defaultIconStyle}
+          />
+          {this.props.activeTool ? this.props.activeTool.replace("_", " ").toLowerCase() : "Move"}
+        </span>
+        {this.props.isSkeletonAnnotation && (
+          <span style={defaultShortcutStyle}>
+            <img
+              className="keyboard-mouse-icon"
+              src="/assets/images/icon-statusbar-mouse-right.svg"
+              alt="Mouse Left"
+              style={defaultIconStyle}
+            />
+            Place Node
+          </span>
+        )}
```

Comment:
This part also needs to be adapted for the current tool (since one cannot place nodes in the brush tool). I'd recommend a simple object which maps from tool to right-click-action-description à la:
```
const rightClickToLabel = {
  MOVE: isSkeletonAnnotation ? "Place Node" : null,
  BRUSH: "Erase",
  TRACE: "Erase",
  FILL_CELL: null,
  PICK_CELL: null,
}
```

that way, we can easily add "open context menu"  as soon as we support that (instead of null).